docs/extensibility/create-extension.md

```diff
@@ -0,0 +1,363 @@
+# Create Extension
+
+This article provides you through process of preparing own extension responsible for ConfigMap management. This show-case extension will provide following functionalities:
+
+* Get ConfigMap from a cluster
+* Create ConfigMap with given name, namespace and data
+* Delete ConfigMap based on its name and namespace
+
+The extension will provides main command (command group) `configmap` that will do nothing instead of printing the `help` on execution, but will has three sub-commands (`create`, `get`, `delete`) with resource-oriented actions described in the list above.
+
+## Steps
+
+1. Prepare ConfigMap with root command
+
+    The very first step will be preparing [a ConfigMap](./README.md#configmap) with required labels and data. For this usecase we would like to have root command `configmap` without any action performed on execution. Let's create ConfigMap with such command and description, following [extensions standards](./README.md#extension-standards), what this command will do:
```

Comment:
"One ConfigMap to rule them all, One ConfigMap to find them, ..." :)
We can do: `$ kyma alpha configmap delete my-extension` :)